src/Services/SwaggerService.php

```diff
@@ -104,13 +109,37 @@ protected function generateEmptyData() {
         return $data;
     }
 
-    protected function getAppUrl() {
+    protected function getAppUrl()
+    {
         $url = config('app.url');
 
         return str_replace(['http://', 'https://', '/'], '', $url);
     }
 
-    protected function generateSecurityDefinition() {
+    /**
+     * @param $info
+     * @return mixed
```

Comment:
Looks like this annotation does not contain helpful information. Could you please remove it?